### ISSUE TITLE
Missing Zope crontab configuration

### ISSUE THREAD

**Issue by pumazi** [Connexions/cnx-deploy#106]
## Summary

The PDF generation zope client is missing configuration necessary to invoke a cron-like job for PDF generation.
## Environment

all

**Comment by pumazi** [comment]
A question about this was posted in the #cnx slack channel. Ross replied with: 

> its in dist_cnx.cfg:
> ```[shared]
# Used below
zeo-host = backend1.cnx.org:7700
# Used by the ClockServer config. See below
portal-name = plone 
zope-user = www-data
clock-server-conf = 
    %import Products.ClockServer
    <clock-server>
        method /${shared:portal-name}/queue_tool/clockTick
        period 3
        user
        password
    </clock-server>
```
> The clock-server stanza ends up in `<buildout>/parts/instance/etc/zope.conf` and runs the Lineup (queue_tool) clockTick method every 3 <timeunits> (Seconds? really?)
> Once you snarfle through all that, the actual PDF building happens with scripts in <buildout>/scripts

**Comment by pumazi** [comment]
Resolved in #141
